### ISSUE TITLE
enable Windows build

### ISSUE THREAD

**Issue by yarikoptic** [conda-forge/git-annex-feedstock#15]
Any clue how difficult that would be? ;)

**Comment by yarikoptic** [comment]
https://github.com/datalad/datalad-extensions/pull/40 might pave the way (although not sure about libmagic since cross-building is probably not allowed, but probably build could just fetch the artifact from datalad-extensions or eventually "official" source).

**Comment by yarikoptic** [comment]
ok -- we have 2 ways to proceed:
- try to reproduce that github actions build recipe (now complete and works!  with libmagic support)
- "cheat": download `git-annex-installer.exe`, `7z x` it and place binaries (if any moving needed) where desired.  I would if that would work (yet to try, but I will).  Note: we are not yet re-distributing those binary builds "publicly" (TODO: https://github.com/datalad/datalad-extensions/issues/46) so we could start with "stock" git-annex
  - known "issue": our builds do not work on windows 7 AFAIK.

**Comment by yarikoptic** [comment]
@notestaff what would be your thinking/desire? ;)

**Comment by notestaff** [comment]
"try to reproduce that github actions build recipe" seems more correct.  Long-term, if something changes in the conda-forge build process, a recipe that does the building itself is more likely to still work / be easier to adjust?  Unfortunately right now I don't have the bandwidth to figure this out, or the knolwedge of building on Windows...

**Comment by yarikoptic** [comment]
yeah, bandwidth is limited here too and I always double think when I impose Windows goodness on someone ;)  So what about two stage
- short term -- "cheat" way, from http://datasets.datalad.org/datalad/packages/windows/ (I am testing such pure "extract" locally in a VM ATM)
- long term -- "proper" build (there is little point to struggle if we see that windows build from "cheat" is "not working").

**Comment by notestaff** [comment]
Sounds good.  This won't be the only recipe to use such a "cheat" until a better way gets worked out.

**Comment by notestaff** [comment]
p.s. what I really wish we had on Windows was a cygwin-based version, but unfortunately GHC has stopped supporting cygwin
